### DISCUSSION TITLE
Change repo visibility to public

### DISCUSSION THREAD

**Discussion by dkomeza** [toasted-games/project-c#9] [Polls]
# Change visibility to public

Advantages:
- Good for repo management, actions, branch protection etc
- Can be good for personal CV
- Support from open-source community?

Disadvantages:
- Potential problems with selling the game in the future?

@toasted-games

**Comment by pvczku** [comment]
I personally think it is a good idea to convert it to public since it is our first serious project and selling the game is not necesarry atm 😃

**Comment by dkomeza** [comment]
With majority of votes the repo will be converted to public.
